### PR TITLE
Created a background, changed fonts, altered list colors, to look mor…

### PR DESCRIPTION
**Pull request by williammadisondavis**
…e appealing. I also changed the title for our locations to be created in HTML as h3 elements, rather than divs. Could y'all review and tell me if you like the background/colors? I loaded into our images folder multiple backgrounds that all look "OK."

### CODE REVIEW

**Comment by clinturbin** [comment]
Since the new image files are to be used for the background, we should probably place them in the assets folder.

File: stylesheets/styles.css [hunk 1]
```diff
@@ -166,15 +166,20 @@ body, html {
 
     .map {
         margin-top: 1em;
-        width: 100%;
-        height: 85%;
+        width: 50%;
+        height: 90%;
+        border-radius: 5px;
     }
```
Comment (running-on-sunshine) [inline stylesheets/styles.css]: Could we maybe change the border-radius from 'px' to 'em' units in order to keep the stylesheet uniform?

**Comment by running-on-sunshine** [comment]
Yes! Agree with @clinturbin re: placing new images into the assets folder because it looks like they are going to be used as a background. 🙂

**Comment by clinturbin** [comment]
Looks good!  OK to merge 👍

**Comment by running-on-sunshine** [comment]
Thoughts:

- Thank you for making updates based on our suggestions! Really appreciate this! 
- Really like the styling changes that you made! Great work! 🙂

When you're ready, definitely feel free to give this a merge! 👍

**Comment by clinturbin** [comment]
Agreed! 👍

**Pull request merged**